website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,176 @@
+See [Well-Known Labels, Annotations and Taints](https://kubernetes.io/docs/reference/labels-annotations-taints/) for details.
+Keep in mind that only a subset of these labels are supported in Karpenter, as described later.
+
+* **Deprovisioning nodes**: A provisioner can also include time-to-live values to indicate when nodes should be deprovisioned after a set amount of time from when they were created or after they becomes empty of deployed pods.
+
+* **Multiple provisioners**: Multiple provisioners can be configured on the same cluster.
+For example, you might want to configure different teams on the same cluster to run on completely separate capacity.
+One team could run on GPU nodes while another uses storage nodes, with each having a different set of defaults.
+
+Although some use cases are addressed witha single provisioner for multiple teams, multiple provisioners are useful to isolate nodes for billing, use different defaults (such as no GPUs for a team), or use different deprovisioning settings.
+
+In the future, multiple provisioners would enable to provision nodes from different cloud providers on the same cluster or even a control plane on one cloud and data plane on another.
+
+### Deprovisioning nodes
+
+Karpenter not only manages the creation of nodes to add cluster capacity, but also will delete nodes when that capacity is no longer needed.
+There are several things you should know about how Karpenter deprovisions nodes:
+
+* **Node Expiry**: If a node expiry time-to-live value (`ttlSecondsUntilExpired`) is reached, that node is drained of pods and deleted (even if it is still running workloads).
+* **Empty nodes**: When the last workload pod running on a Karpenter-managed node is gone, a clock starts keeping track of how long the node is empty.
+Once that "node empty" time-to-live (`ttlSecondsAfterEmpty`) is reached, that node is deleted.
+* **Finalizer**: Karpenter places a finalizer bit on each node it creates.
+When a request comes in to delete one of those nodes (from an expired TTL or maybe an explicit `kubectl delete node`), Karpenter will drain all the pods and then terminate the EC2 instance.
+Karpenter handles all clean-up work needed to properly delete the node.
+
+For more details on how Karpenter deletes nodes, see [Deleting nodes with Karpenter](/docs/tasks/delete-nodes.md) for details.
+
+### Upgrading nodes
+
+A straight-forward way to upgrade nodes that are managed by Karpenter is to add new upgraded nodes, move pod workloads from the old nodes to the new ones, and delete the old nodes.
+Another way to do upgrades is to set `ttlSecondsUntilExpired` values so old nodes are deleted after a set period of time, so they can then be replaced with newer nodes.
+For details on upgrading nodes with Karpenter, see [Upgrading nodes with Karpenter](/docs/tasks/upgrade-nodes.md) for details.
+
+### Cluster operator concepts
```

Comment:
This h3 reads a bit odd to me since we're already in the h2 Cluster Operator. Perhaps simpler to promote each of these h4s to h3.
```suggestion
```